testspace/testspace.py

```diff
@@ -68,7 +64,12 @@ def push(
         if message is not None:
             command_args_list.append("--message={}".format(message))
 
-        subprocess.run(command_args_list, check=True)
+        subprocess.run(
+            " ".join(command_args_list),
```

Comment:
shell=True handles the list passed as a command differently than previous.
https://medium.com/python-pandemonium/a-trap-of-shell-true-in-the-subprocess-module-6db7fc66cdfd